PocketKit/Package.swift

```diff
@@ -25,16 +25,16 @@ let package = Package(
         .library(name: "DiffMatchPatch", targets: ["DiffMatchPatch"])
     ],
     dependencies: [
-        .package(url: "https://github.com/apollographql/apollo-ios.git", from: "1.9.0"),
+        .package(url: "https://github.com/apollographql/apollo-ios.git", from: "1.12.2"),
         .package(url: "https://github.com/onevcat/Kingfisher.git", from: "7.11.0"),
         .package(url: "https://github.com/getsentry/sentry-cocoa.git", from: "8.22.4"),
-        .package(url: "https://github.com/snowplow/snowplow-objc-tracker", from: "6.0.1"),
-        .package(url: "https://github.com/ccgus/fmdb", from: "2.7.6"),
-        .package(url: "https://github.com/airbnb/lottie-ios.git", from: "4.4.0"),
+        .package(url: "https://github.com/snowplow/snowplow-objc-tracker", from: "6.0.3"),
+        .package(url: "https://github.com/ccgus/fmdb", from: "2.7.11"),
```

Comment:
it shouldn't be necessary to update the minors since we have the `from version` parameter. On the other hand it won't hurt

Reply:
@Gio2018 yep aware, but the lock file was out of date, and this will help remove them from the dep issue.